flag.go

```diff
@@ -94,6 +94,7 @@ type Flag interface {
 	Apply(*flag.FlagSet) error
 	Names() []string
 	IsSet() bool
+	RunAction(*Context) error
```

Comment:
This would break all existing flag implementations. So usually we wrap this into a separate interface say 

type ActionableFlag interface {
   Flag
   RunAction(*Context) error
}